platform/rct/hal/usb_stm32/USB-CDC_device_stm32.c

```diff
@@ -1,52 +1,250 @@
-#include "USB-CDC_device.h"
```

Comment:
I respectfully disagree.  You sort includes so that people can easily find if includes are there or not and to be able to easily see if someone has duplicated the includes.  If code is effected by order of includes, then exposing this flaw during development is very important to do so that it can be fixed.  Order of includes should never matter in C so long as they are all at the top of the file.  Ensuring they are all there and not duplicated is why we do it.